### ISSUE TITLE
"Invalid severity Information" after launch

### ISSUE THREAD

**Issue by soniccat** [passy/android-lint-summary#20]
After passing `./gradlew lintAndroidMarketDebug` and launching `android-lint-summary` I have only that on a project with 33 errors and 615 warnings.

```
[F] Incomplete translation
    /Users/alexeyglushkov/androidProjects/../main/res/values/strings.xml:20
[F] Incomplete translation
    /Users/alexeyglushkov/androidProjects/.../main/res/values/strings.xml:21
[F] Incomplete translation
    /Users/alexeyglushkov/androidProjects/../main/res/values/strings.xml:46
android-lint-summary: Invalid severity Information
```

Are there any way to fix it?

**Comment by passy** [comment]
@soniccat Thanks for reporting this. Could you perhaps send me your `lint-results.xml` file that causes this so I can include this as a test case?

**Comment by soniccat** [comment]
@passy yes, I've sent the xml to your email passy@twitter.com. The xml is the result of analyzing Facebook sdk.

**Comment by passy** [comment]
@soniccat Thank you!

**Comment by passy** [comment]
Fixed with `0.3.2` :)
